utils/fileutils/fileutils_test.go

```diff
@@ -14,175 +14,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestFindConfigFile(t *testing.T) {
```

Comment:
These tests were just duplicates of `fileutils.FindFile` but from the `config/` perspective. There's already coverage for this in the file store.